projects/components/src/assets/resources/en.properties

```diff
@@ -73,6 +73,7 @@ vcd.cc.numeric.filter.to=To:
 vcd.cc.datagrid.actions=Actions
 
 vcd.cc.action.menu.actions=All Actions
+vcd.cc.action.menu.other.actions=Other Actions
```

Comment:
Because this fix is being merged into master and that is the branch that will be used in 10.2.1 (is this true still?), are there any i18n restrictions for patch releases? Is there anything extra we need to do via communicating this to Sarah?

Reply:
We'll have a separate branch for VCD's 10.2.1, which we'll call a11y here